tools/readahead.py

```diff
@@ -89,15 +92,74 @@
 }
 """
 
-b = BPF(text=program)
-if BPF.get_kprobe_functions(b"__do_page_cache_readahead"):
-    ra_event = "__do_page_cache_readahead"
+bpf_text_kfunc = """
+KFUNC_PROBE(RA_FUNC, void *unused)
```

Comment:
Add an `unused` argument here since the KFUNC_PROBE macro does NOT allow empty arguments.

Maybe it need a `##` ?
```c
#define KFUNC_PROBE(event, args...) \
        BPF_PROG(kfunc__ ## event, ##args)
```